Model/Hotspot/Hotspot.swift

```diff
@@ -21,9 +21,9 @@ final class Hotspot {
     @MainActor
     static let shared = Hotspot()
     
-    private static let minPort = 1024
+    private static let minPort = 1
```

Comment:
I'm skeptical about this. Usually (and even historically on macOS) users cant open ports under 1024 (the reason you started from there). I see that on my computer, I can open such ports as a regular user but I don't know if that's something I did. I doubt it's possible on iOS but I didn't check. Pretty sure older versions of macOS/iOS wouldn't support it without some privilege excalation.

Reply:
@rgaudin To articulate even a bit more about my request, I just want to give freedom - as far as reasonable - to the user and not arbitrary put a preset range of allowed ports. If possible, to me, `80` is a very valid port. This is why I have asked about the "rationals".

Reply:
Indeed, I think a privileged-less default is fine and allowing any port (via escalation if required) would be great. If it happens that all the platforms we support allow binding to 80 without intervention, then defaulting to 80 would be even better